Tofu/AccountCell.swift

```diff
@@ -80,7 +80,7 @@ private func imageForAccount(_ account: Account) -> UIImage {
     case .some("Microsoft"): return UIImage(named: "Microsoft")!
     case .some("Name.com"): return UIImage(named: "Name.com")!
     case .some("Nextcloud"): return UIImage(named: "Nextcloud")!
-    case .some("NiceHash - New platform"): return UIImage(named: "NiceHash")!
+    case .some("NiceHash"): return UIImage(named: "NiceHash")!
```

Comment:
What do you think about supporting both issuer strings? This would allow people who set up 2FA for NiceHash before the change, and never changed the Issuer themselves, to keep the icon.

```suggestion
    case .some("NiceHash"), .some("NiceHash - New platform"): return UIImage(named: "NiceHash")!
```